src/app/state/transaction/saga.ts

```diff
@@ -294,13 +325,35 @@ export function* submitParaTimeTransaction(runtime: Runtime, transaction: ParaTi
     nic,
     paraTimeTransactionSigner as Signer,
     transaction,
-    fromAddress,
+    fromOasisAddress,
     runtime,
   )
 
   yield* call(OasisTransaction.signParaTime, chainContext, paraTimeTransactionSigner as Signer, rtw)
   yield* call(OasisTransaction.submit, nic, rtw)
   yield* put(transactionActions.paraTimeTransactionSent(transaction.recipient))
+
+  const hash = yield* call(hashSignedTransaction, rtw.unverifiedTransaction as any) // TODO use rtw.hash when added
```

Comment:
@oasisprotocol/client-rt doesn't seem to have a way to get runtime transaction hash. This works at least for deposit and withdraw

cc @pro-wh